src/Aquifer.API/Endpoints/Marketing/Unsubscribe/Endpoint.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Aquifer.Common.Utilities;
+using Aquifer.Data;
+using FastEndpoints;
+
+namespace Aquifer.API.Endpoints.Marketing.Unsubscribe;
+
+public class Endpoint(AquiferDbContext dbContext, ILogger<Endpoint> logger) : Endpoint<Request>
+{
+    public override void Configure()
+    {
+        Get("/marketing/unsubscribe/{UnsubscribeId}");
+        AllowAnonymous();
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        try
+        {
+            var subscriberToDisable = dbContext.ContentSubscribers.SingleOrDefault(s => s.UnsubscribeId == req.UnsubscribeId);
+            if (subscriberToDisable is not null)
+            {
+                subscriberToDisable.Enabled = false;
+                await dbContext.SaveChangesAsync(ct);
+            }
+        }
+        catch(Exception e)
+        {
+            logger.LogError(e, "Failed to handle subscriber request: {requestContent}", JsonUtilities.DefaultSerialize(req));
+        }
```

Comment:
Do we need this catch here? In most places we aren't doing try catch blocks unless we're reaching out to an external API or something.

Reply:
I had seen it in the `marketing/subscribers` endpoint, and then BIB-708 says to send OK no matter what, so I had thought maybe having something logged somewhere would be needful. Not sure if that's good enough reasoning though.

Reply:
I think the SingleOrDefault shouldn't blow up if it doesn't find something. It'll just skip past the if != null block. Just doing the SendOk at the end without try/catch should be fine.